documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -4,7 +4,8 @@
 
 ### Changed features
 
-* The `logging.level.com.synopsys.integration` property is being removed in [company_name] [solution_name] 10.0.0. Please use the `logging.level.detect` property instead. There is no difference between the two properties.
+* The `logging.level.com.synopsys.integration` property will be deprecated in favor of the `logging.level.detect` property in [company_name] [solution_name] 10.0.0. 
```

Comment:
Deprecated in 9.10.0 and removed in 10.0.0

Reply:
Ah yes, my bad! Thank you. 
New MR for that.